neurokit2/ecg/ecg_process.py

```diff
@@ -85,34 +89,31 @@ def ecg_process(ecg_signal, sampling_rate=1000, method="neurokit"):
       plt.close()
 
     """
-    # Sanitize input
-    ecg_signal = signal_sanitize(ecg_signal)
 
+    # Sanitize and clean input
+    ecg_signal = signal_sanitize(ecg_signal)
     ecg_cleaned = ecg_clean(ecg_signal, sampling_rate=sampling_rate, method=method)
-    # R-peaks
-    (instant_peaks, rpeaks,) = ecg_peaks(
-        ecg_cleaned=ecg_cleaned, sampling_rate=sampling_rate, method=method, correct_artifacts=True
-    )
 
+    # Detect R-peaks
+    instant_peaks, rpeaks = ecg_peaks(ecg_cleaned, sampling_rate, method, correct_artifacts=True)
+
+    # Calculate heart rate
     rate = signal_rate(rpeaks, sampling_rate=sampling_rate, desired_length=len(ecg_cleaned))
 
+    # Assess signal quality
     quality = ecg_quality(ecg_cleaned, rpeaks=rpeaks["ECG_R_Peaks"], sampling_rate=sampling_rate)
 
-    signals = pd.DataFrame(
-        {"ECG_Raw": ecg_signal, "ECG_Clean": ecg_cleaned, "ECG_Rate": rate, "ECG_Quality": quality}
-    )
+    # Merge signals in a DataFrame
+    signals = pd.DataFrame({"ECG_Raw": ecg_signal, "ECG_Clean": ecg_cleaned, "ECG_Rate": rate, "ECG_Quality": quality})
 
-    # Additional info of the ecg signal
-    delineate_signal, delineate_info = ecg_delineate(
-        ecg_cleaned=ecg_cleaned, rpeaks=rpeaks, sampling_rate=sampling_rate
-    )
+    # Delineate QRS complex
+    delineate_signal, delineate_info = ecg_delineate(ecg_cleaned, rpeaks, sampling_rate)
```

Comment:
any reason to remove keyword arguments?

Reply:
I removed the keyword arguments every time the keyword name is the same as the name of the variable passed.

From a technical perspective, positional arguments work just as well, so there is no need for using the keywords.

From a code comprehensibility perspective, I find it unnecessary to have the same information twice. If the variable name already states what it is, then the keyword name is not needed – just makes the line longer and harder to read IMHO.

But if for some reason keyword arguments are preferred in NeuroKit, I can also change this back. I just though brevity might make the code easier to read.

Reply:
Fine by me, I have no strong preference here, was just curious about the motivation

Reply:
I'd prefer using keywords arguments. I don't like relying on order and I like to see what goes in which argument (although the kwargs and the vars have the same name, but in general it's good to keep consistency)

Reply:
Understood and duly noted for future contributions. I reverted the changes in question to use keyword args instead of positional args.